cfpb/consumerfinance.gov
# Update to Design System 0.7.9

## Changes

- Update to Design System 0.7.9 dependencies.


## How to test this PR

1. Run `./frontend.sh`
2. Visit http://localhost:8000/about-us/blog/ and see that Topics and Authors labels are linked to their multiselects. 
3. Visit http://localhost:8000/ask-cfpb/how-can-i-tell-if-a-friend-neighbor-or-family-member-is-a-victim-of-financial-exploitation-en-1933/ at mobile size, and if changing the font size in Chrome settings (`cmd + ,` > `Font size` > `Very large`) the summary components summary area is 88px regardless of font size.
